src/metabase/cmd/endpoint_dox.clj

```diff
@@ -5,22 +5,120 @@
             [metabase.plugins.classloader :as classloader]
             [metabase.util :as u]))
 
+;;;; API docs intro
+
+(defn- api-docs-intro!
+  "Exists just so we can write the intro in Markdown."
+  []
+  (str (slurp "src/metabase/cmd/resources/api-intro.md") "\n\n"))
+
+;;;; API docs section title
+
+(defn- endpoint-ns-name
+  "Creates a name for endpoints in a namespace, like all the endpoints for Alerts."
+  [endpoint]
+  (->> (:ns endpoint)
+       (ns-name)
+       (name)
+       (#(str/split % #"\."))
+       (last)
+       (str/capitalize)))
```

Comment:
```suggestion
  (->> (:ns endpoint)
       ns-name
       name
       (#(str/split % #"\."))
       last
       str/capitalize))
```

You don't need to wrap things in `()` in the threading macros if they don't have any arguments, the `()` happens automatically